src/components/BookmarkManager.vue

```diff
@@ -0,0 +1,180 @@
+<script setup lang="ts">
+import { ref, computed, onMounted } from 'vue'
+
+interface Bookmark {
+  url: string
+  name: string
+  isRenaming: boolean
+}
+
+const bookmarks = ref<Bookmark[]>([])
+
+onMounted(() => {
+  requestBookmark()
+})
+
+const requestBookmark = () => {
+  const response = window.prompt('VERSO::BOOKMARK_LIST_GET')
+  if (response !== null) {
+    const status = JSON.parse(response) as { [url: string]: Bookmark }
+    bookmarks.value = Object.values(status)
+  }
+}
+
+const onclickRemove = (id: string) => {
+  window.prompt('VERSO::BOOKMARK_REMOVE::' + id)
+}
+
+const EndRename = (id: string, name: string) => {
```

Comment:
The bookmark name is a input form user, we should sanitized the input like escaping HTML content or at least escape when we use it in the vue template.

https://stackoverflow.com/questions/6234773/can-i-escape-html-special-chars-in-javascript
https://vuejs.org/guide/best-practices/security.html